src/common/low_precision_transformations/include/low_precision/propagate_to_input.hpp

```diff
@@ -47,26 +47,70 @@ class ngraph::pass::low_precision::PropagateToInput : public ngraph::pass::Match
 
             {
                 OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::LPT_LT, "PropagateToInput");
-
-                for (auto input : node->inputs()) {
-                    auto parentAttribute = getSourceOutputAttribute(input, defaultPrecisions);
-                    if (parentAttribute == nullptr) {
+                std::vector<std::vector<size_t>> groups;
```

Comment:
I would say the code before changes was quite well understandable: it just merged rt info from all the parents. 
Current logic is much more complex: needs some comments with explanation.